README.md

```diff
@@ -45,3 +41,8 @@ with the help of [this article](https://medium.com/dailyjs/running-mocha-tests-a
 The easiest way to run the tests is to launch a local server and open
 `tests.html` in your browser.
 
+## Sound credits
```

Comment:
This is fine for now, but it would be nice if we could eventually make our own sound effects to keep the "made from scratch" theme

Reply:
Yeah, I have a couple ideas to synthesize our own sounds. Definitely temporary.